sinks/gcmautoscaling/driver.go

```diff
@@ -263,7 +265,10 @@ func init() {
 	extpoints.SinkFactories.Register(CreateGCMScalingSink, "gcmautoscaling")
 }
 
-func CreateGCMScalingSink(_ string, _ map[string][]string) ([]sink_api.ExternalSink, error) {
+func CreateGCMScalingSink(uri *url.URL) ([]sink_api.ExternalSink, error) {
+	if *uri != (url.URL{}) {
+		return nil, errors.New("gcmautoscaling sinks don't take arguments")
```

Comment:
Even if they don't have any formatting? I'm guessing this is to be consistent and to not add the `errors` import, correct?